bool initiate_ignition
bool force
